libs/full/segmented_algorithms/tests/unit/partitioned_vector_max_element2.cpp

```diff
@@ -0,0 +1,86 @@
+//  Copyright (c) 2020 Akhil J Nair
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/config.hpp>
+#if !defined(HPX_COMPUTE_DEVICE_CODE)
+#include <hpx/hpx_main.hpp>
+#include <hpx/include/parallel_minmax.hpp>
+#include <hpx/include/partitioned_vector.hpp>
```

Comment:
For the errors about `hpx::find_all_localities` not being defined, please include `hpx/runtime.hpp` here (and in the other tests that need it).

Reply:
Oh ok, thanks